Use `cosmic-client-toolkit` helpers in workspace applet

ids1024 opened the pull request:
Needs some testing to make sure it is working as well as the previous version. (The compositor seems to be reporting somewhat funny workspace information in my current session: https://github.com/pop-os/cosmic-comp/issues/49)

wash2 commented:
Awesome, I've been meaning to do this, and the new app-list applet is using cosmic-client-toolkit as well :) I'll test this out a bit

ids1024 commented:
Rebased, addressed conflicts, and the issue this hard, so this should be good now.

Strangely I had to move `OutputState::new` to before the workspace manager is bound, or it didn't get workspaces associated with the workspace groups. That's expected with `WorkspaceState::new`, but the workspace manger shouldn't be relevant...

ids1024 commented:
Oh, this is binding a `ZcosmicWorkspaceManagerV1`, but that's being dispatched with the cctk code. Which is getting things mixed up.

What we need is for cctk to expose it's manger instance so we don't need to create a new one.
